ihatemoney/web.py

```diff
@@ -895,12 +896,23 @@ def strip_ip_addresses():
 @main.route("/<project_id>/statistics")
 def statistics():
     """Compute what each member has paid and spent and display it"""
-    today = datetime.now()
+    if g.project.has_bills():
+        first_date = g.project.get_least_recent_bill().date
+        last_date = g.project.get_most_recent_bill().date
```

Comment:
first and last doesn't really work for me when dealing with dates. I would prefer if we used `oldest_date` and `newest_date` I guess.